cmd/otelcol/Dockerfile.windows

```diff
@@ -26,8 +26,21 @@ RUN Add-Type -AssemblyName System.IO.Compression.FileSystem ;\
     [System.IO.Compression.ZipFile]::ExtractToDirectory('SignalFxAgent-win64.zip', '.')
 RUN Rename-Item -Path "SignalFxAgent" -NewName "agent-bundle"
 
+# Download JMX Metric Gatherer
+ARG JMX_LIB_VERSION
+RUN New-Item -Path "C:\\" -Name "opt" -ItemType "directory"
+RUN Invoke-WebRequest -Uri "https://repo1.maven.org/maven2/io/opentelemetry/contrib/opentelemetry-jmx-metrics/$env:JMX_LIB_VERSION/opentelemetry-jmx-metrics-$env:JMX_LIB_VERSION.jar" -Outfile "c:\opt\opentelemetry-java-contrib-jmx-metrics.jar"
+
+ARG JAVA_VERSION="11-openjdk-11.0.13.8-1"
+ARG JAVA_ZIP_VERSION="11-openjdk-jre-11.0.13.8-1"
+ARG JAVA_HOME="C:\java\java-${JAVA_ZIP_VERSION}.windows.ojdkbuild.x86_64\bin"
+RUN Invoke-WebRequest -Uri "https://github.com/ojdkbuild/ojdkbuild/releases/download/java-$env:JAVA_VERSION/java-$env:JAVA_ZIP_VERSION.windows.ojdkbuild.x86_64.zip" -OutFile "openjdk.zip" ; \
```

Comment:
Please use something like AdoptOpenJDK which is part of the eclipse foundation: https://github.com/signalfx/signalfx-agent/blob/303b6fdd4b205273c7859c3ccba8b78f94487661/Dockerfile#L48

Reply:
@rmfitzpatrick Changes made to download and extract AdoptOpenJDK for Windows.
Thanks for the correction!